assets/js/wp-hours-widget.js

```diff
@@ -0,0 +1,62 @@
+( function ( $ ) {
+
+	var hoursWidget = {
+
+		init: function( e ) {
+
+			var date        = new Date(),
+					cur_time    = hoursWidget.getCurrentTime( date ),
+					time_blocks = wpcw_hours.schedule[ date.getDay() ].open,
+					$open_sign  = $( '.wpcw-open-sign' );
+
+
+			if ( ! time_blocks ) {
+
+				$open_sign.addClass( 'closed' ).html( wpcw_hours.closed_string );
```

Comment:
@EvanHerman I think we can just use the `text()` method on these instead of `html()`.